zkp/js/test/nf_anon_nullifier.js

```diff
@@ -194,9 +194,9 @@ describe("main circuit tests for Zeto non-fungible tokens with anonymity using n
       err = e;
     }
     // console.log(err);
-    expect(err).to.match(/Error in template Zeto_319 line: 76/);
+    expect(err).to.match(/Error in template Zeto_319 line: 55/);
     expect(err).to.match(
-      /Error in template CheckHashesForTokenIdAndUri_88 line: 58/,
+      /Error in template CheckHashesForTokenIdAndUri_88 line: 51/,
```

Comment:
did it odd that we patten match on line number...

Reply:
@Chengxuan thoughts on this?

Reply:
this is by far the most reliable way to determine the error. Maybe able to find some alternative along the way.